app/src/main/res/layout/project_search_result_view.xml

```diff
@@ -34,27 +35,28 @@
     <LinearLayout
       android:orientation="vertical"
       android:layout_marginEnd="@dimen/grid_2"
-      android:layout_marginRight="@dimen/grid_2"
-      android:layout_centerVertical="true"
-      android:layout_toLeftOf="@id/project_image_view"
-      android:layout_toStartOf="@id/project_image_view"
+      android:layout_alignParentBottom="@id/project_image_view"
+      android:layout_marginStart="@dimen/grid_2"
```

Comment:
same here, should have accompanying `marginLeft` and `marginRight` since you have start and end both defined here